.github/workflows/e2e.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest-4-cores
     env:
       # the gh tag of system-test repo version to run
-      SYSTEM_TEST_GIT_REF: 352cc1d4ea96af8cfb2c2b7335c3b51da000f889
+      SYSTEM_TEST_GIT_REF: master
```

Comment:
yeah, seems fine to just ref the latest, if it gets too far ahead, then can come back and pin it here.